ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2475,3 +2475,35 @@ def test_tensorflow_greater_equal(
         x=x[0],
         y=x[1],
     )
+    
+    
+    
+    
+# in_top_k
+@handle_frontend_test(
+    fn_tree="tensorflow.math.in_top_k",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

Let's get all the valid dtypes not just the numeric ones